mev_inspect/traces.py

```diff
@@ -1,80 +1,33 @@
-from itertools import groupby
-from typing import Iterable, List
+from typing import List
 
-from mev_inspect.schemas import Trace, NestedTrace
+from mev_inspect.schemas.classified_traces import ClassifiedTrace
 
 
-def as_nested_traces(traces: Iterable[Trace]) -> List[NestedTrace]:
-    nested_traces = []
+def is_child_trace_address(
```

Comment:
Yeah that's fair to say. Nested trace object basically just isn't used. I've at least found it easier to look at the traces linearly rather than reconstructing the tree. Might be a future use case where we want to bring the tree back though.